apollo-portal/src/main/resources/static/scripts/controller/SystemInfoController.js

```diff
@@ -28,13 +28,13 @@ function SystemInfoController($scope, toastr, AppUtil, AppService, ClusterServic
         });
     }
 
-    function check(host) {
-        SystemInfoService.check_health(host).then(function (result) {
+    function check(instanceId) {
+        SystemInfoService.check_health(instanceId).then(function (result) {
             var status = result.status.code;
-            if (status == 'UP') {
-                toastr.success(host + ' is healthy!');
+            if (status === 'UP') {
+                toastr.success(instanceId + ' is healthy!');
             } else {
-                toastr.error(host + ' is not healthy, please check ' + host + '/health for more information!');
+                toastr.error(instanceId + ' is not healthy, please check /health of ' + instanceId + 'for more information!');
```

Comment:
I think the host is more user friendly in the error message. How about we pass both host and instanceId into this function and use instanceId to request the backend while using host in the toaster?